src/annotator/sidebar.tsx

```diff
@@ -43,6 +44,9 @@ export type SidebarConfig = { sidebarAppUrl: string } & Record<string, unknown>;
  * Client configuration used by the sidebar container ({@link Sidebar}).
  */
 export type SidebarContainerConfig = {
+  /** CSS selector for the container of the bucket bar. */
+  bucketBarContainer?: string;
```

Comment:
It might make sense to accept an element here as well in future. A CSS selector as it is JSON-serializable, for use in eg. the video player which uses JSON rather than JS config.

Reply:
Should this perhaps be named `bucketContainerSelector` or somesuch to signal its type and meaning better?